example/src/App.tsx

```diff
@@ -20,6 +25,12 @@ export default function App() {
     number | undefined
   >();
   const [steps, setSteps] = useState<number | undefined>();
+  const [activeEnergyBurned, setActiveEnergyBurned] = useState<
+    number | undefined
```

Comment:
You can omit "undefined" if the default value is empty

```suggestion
    number
```

Reply:
Feel free to ignore this message if you prefer to make the "undefined" type explicit